Firestore/Swift/Source/Stages.swift

```diff
@@ -26,6 +26,13 @@ import Foundation
 protocol Stage {
   var name: String { get }
   var bridge: StageBridge { get }
+  var errorMessage: String? { get }
```

Comment:
Let's document the purpose and the behavior of this field in detail, because this is fairly ununsual.